packages/mattermost-redux/src/actions/users_queries.ts

```diff
@@ -78,33 +80,34 @@ query gqlWebCurrentUserInfo {
 
 export const currentUserInfoQuery = JSON.stringify({query: currentUserInfoQueryString, operationName: 'gqlWebCurrentUserInfo'});
 
+type GraphQlUser = UserProfile & {
```

Comment:
I didn't realize that we were capitalizing "QL" this way here. I guess it fits with the pattern of how we capitalize initialisms like ID, but reading `GraphQl` feels weird to me

Reply:
yes changed